test-network-function/lifecycle/suite.go

```diff
@@ -104,9 +108,65 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 		}
 
 		testOwner(&configData)
+
+		testScaling(&configData)
 	}
 })
 
+func waitForAllDeploymentsReady(namespace string, timeout, pollingPeriod time.Duration) {
+	gomega.Eventually(func() []string {
+		_, notReadyDeployments := getDeployments(namespace)
+		return notReadyDeployments
+	}, timeout, pollingPeriod).Should(gomega.HaveLen(0))
+}
+
+func testScaling(configData *common.ConfigurationData) {
+	ginkgo.It("Testing deployment scaling", func() {
+		defer results.RecordResult(identifiers.TestScalingIdentifier)
+
+		// Map to register the deployments that have been already tested
+		deploymentNames := make(map[string]bool)
+
+		for _, cut := range configData.ContainersUnderTest {
+			namespace := cut.Oc.GetPodNamespace()
+
+			// Get deployment name and check whether it was already tested.
+			// ToDo: Proper way (helper/handler) to do this.
+			podNameParts := strings.Split(cut.Oc.GetPodName(), "-")
+			deploymentName := podNameParts[0]
+
+			if _, alreadyTested := deploymentNames[deploymentName]; alreadyTested {
```

Comment:
where is deploymentNames[] updated with the already tested ?

Reply:
@greyerof Just FYI, we are going to add deploymentUnderTest to the config api soon.

Reply:
@edcdavid It is updated in line 165.
@jun-rh Ok, I'll update it as soon as that new api is available.

Reply:
@greyerof ok got it thx